.github/workflows/test-python.yml

```diff
@@ -67,8 +67,8 @@ jobs:
 
```

Comment:
it looks like the line above this is incorrect

Reply:
You mean `uv sync --no-install-package geoindex-rs` should be just `uv sync`?

Reply:
no, it needs to be `uv sync --no-install-package async-tiff`. And then the lines after that still need the `--no-project`.

You see this line: https://github.com/developmentseed/async-tiff/actions/runs/14786441598/job/41515604193#step:7:19

When you run `uv sync` it _builds the rust code in release mode_ as part of the current project. For fastest CI we want to build and test in debug mode, not release mode. That means we want to tell `uv` **not** to automatically build `async-tiff`, because we'll manually build it after with `maturin develop`

Reply:
Ok, I think you're right. Checking the timings:

- Commit 12b230f57c67b7bb02105e3cc235d194c7c6d647, job https://github.com/developmentseed/async-tiff/actions/runs/14783290369/job/41506715402:
  - `uv sync` = 2m0s
  - `uv run maturin develop` = 46s
  - Total: 2m46s
- Commit 638d6029b03fe15262386ceb98015514ed8579d8, job https://github.com/developmentseed/async-tiff/actions/runs/14786316208/job/41515269407:
  - `uv run maturin develop` = 2m54s

So let's do `uv sync --no-install-package async-tiff` && `uv run --no-project maturin develop`.

Reply:
Final timings at commit aa213688c943406f8a2d9eaa6b4fcd7347857210, job https://github.com/developmentseed/async-tiff/actions/runs/14825301405/job/41617636468:
- `uv sync --no-install-package async-tiff` = 3s
- `uv run --no-project maturin develop` = 46s
- Total: 49s :rocket: